### PR TITLE
Fix ActiveState getting started Trusted Publisher links

### PR DESCRIPTION
**Pull request by th3coop**

### CODE REVIEW

**Comment by th3coop** [comment]
Huh...I'm not sure how there could be a conflict.  I made this branch from the upstream/more-oidc-docs branch.

File: docs/user/trusted-publishers/adding-a-publisher.md [hunk 1]
```diff
@@ -99,8 +99,10 @@ each.
 
     For ActiveState, you must provide the name of the ActiveState project,
     the ActiveState organization that project belongs to, and the ActiveState user performing
+
```
Comment (di) [inline docs/user/trusted-publishers/adding-a-publisher.md:102]:
```suggestion
```

File: docs/user/trusted-publishers/creating-a-project-through-oidc.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ provide the name of the PyPI project that will be created.
     Setting up ActiveState to create a PyPI project is the same as
     updating a project. You need to provide the name of the ActiveState project,
     the ActiveState organization that project belongs to, and the ActiveState user who will be performing
-    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
+    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/pypi).
```
Comment (di) [inline docs/user/trusted-publishers/creating-a-project-through-oidc.md:68]: This is 404 as well, is that expected?

Comment (di) [reply docs/user/trusted-publishers/creating-a-project-through-oidc.md:68]: Looks like this should be:
```suggestion
    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/pypi/).
```

File: docs/user/trusted-publishers/security-model.md [hunk 3]
```diff
@@ -170,7 +170,7 @@ own security model and considerations.
     * The ActiveState Platform project must be private.
 
     For more information about Trusted Publishing using the ActiveState Platform please
-    see the [PyPI configuration documentation](https://docs.activestate.com/platform/start/PYPI) and the [ActiveState Platform documentation](https://docs.activestate.com/platform).
+    see the [PyPI configuration documentation](https://docs.activestate.com/platform/start/pypi) and the [ActiveState Platform documentation](https://docs.activestate.com/platform).
```
Comment (di) [inline docs/user/trusted-publishers/security-model.md:173]:
```suggestion
    see the [PyPI configuration documentation](https://docs.activestate.com/platform/start/pypi/) and the [ActiveState Platform documentation](https://docs.activestate.com/platform).
```

File: docs/user/trusted-publishers/creating-a-project-through-oidc.md (hunk 2 above)
Comment (th3coop) [reply docs/user/trusted-publishers/creating-a-project-through-oidc.md:68]: It's working for me...I see you suggested a change below to add the trailing slash though...and now that I look, my browser ADDS that 🤦 .  Fixing.

File: docs/user/trusted-publishers/adding-a-publisher.md (hunk 1 above)
Comment (th3coop) [reply docs/user/trusted-publishers/adding-a-publisher.md:102]: Done locally.

File: docs/user/trusted-publishers/adding-a-publisher.md [hunk 4]
```diff
@@ -99,8 +99,10 @@ each.
...
     the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
```
Comment (rawktron) [inline docs/user/trusted-publishers/adding-a-publisher.md:103]: Weird, the change that was on line 103 here that fixes this URL looks to have gotten wiped out @th3coop

**Review by rawktron — CHANGES_REQUESTED** [review]

File: docs/user/trusted-publishers/adding-a-publisher.md [hunk 5]
```diff
@@ -99,8 +99,9 @@ each.
 
     For ActiveState, you must provide the name of the ActiveState project,
     the ActiveState organization that project belongs to, and the ActiveState user performing
-    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
+    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/pypi/).
     ![Image showing adding a new ActiveState publisher](/assets/trusted-publishing/activestate/project-publishing-form.png)
+
```
Comment (th3coop) [inline docs/user/trusted-publishers/adding-a-publisher.md:104]: DOH.  Getting rid...

**Review by di — APPROVED** [review]
We got there eventually 😅

File: docs/user/trusted-publishers/adding-a-publisher.md (hunk 4 above)
Comment (th3coop) [reply docs/user/trusted-publishers/adding-a-publisher.md]: There was a merge conflict.  I must have brought it back fixing that conflict

**Pull request merged**